web/fabfile.py

```diff
@@ -305,4 +305,33 @@ def casebook_garbage_collect(older_than_days=180, dry_run=False):
         if not dry_run:
             cb.delete()
         count += 1
-    print(f"Deleted {count}/{len(cbs)} previous saves older than {older_than_days} days old")
\ No newline at end of file
+    print(f"Deleted {count}/{len(cbs)} previous saves older than {older_than_days} days old")
+
+
+@task
+@setup_django
+def export_node(node_id=None, casebook_id=None, ordinals=None, annotations=True, file_name="temporary_export.docx"):
+    from time import time
+    from main.models import ContentNode
+    content_node = None
+    if node_id:
+        content_node_id = int(node_id)
+        content_node = ContentNode.objects.get(id=content_node_id)
+    else:
+        casebook_id = int(casebook_id)
+        ords = list(map(int, ordinals.split(".")))
+        content_node = ContentNode.objects.get(casebook_id=casebook_id, ordinals=ords)
+    if not content_node:
```

Comment:
`ContentNode.objects.get` will raise `main.models.ContentNode.DoesNotExist` if no such node exists, so I think you want a `try`/`except` here for error handling rather than `content_node = None` and `if not content_node`

Reply:
Changed.